### DISCUSSION TITLE
MvxSplashScreenActivity InitializationComplete removed

### DISCUSSION THREAD

**Discussion by jbouche-work** [MvvmCross/MvvmCross#4850] [Q&A] [answered]
Hi,

I'm migrating MvvmCross 8.0.2 to 9.2.0.

The InitializationComplete() method was removed of the new SplashScreeActivity since it inherit of MvxStartActivity instead of MvxSplashScreenActivity.
I have a lot of stuff initialized (Crashlytics, notifications, navigation by scheme, ...) in the removed InitializationComplete() method.

Can you recommend me where I can now do that initialization stuff ?

In another method of the MvxStartActivity  ?
In a method of the MvxAndroidApplication I suppose ?

Thanks in advance.

**Comment by Cheesebaron** [comment, marked as answer]
Crashlytics -> MainAndroidApplication/Main Activity
Notifications -> Main Activity
Navigation by scheme -> ? No idea what this is

But relying on a 3rd party framework to initialize fundamentals in your App seems like a bad design choice to me

**Reply by jbouche-work** [reply]
Hi, Thanks for your feedback.
